README.md

```diff
@@ -41,19 +40,19 @@ The key/value pairs insides this ConfigMap are:
 
 The reference's location and format depends on the following scenarios:
 
-1. OLM-enabled Operator: Use the `statusDescriptor` part of the CSV to mark which `status` properties reference the binding data:
+1. OLM-enabled Operator: Use the `statusDescriptor` part of the CSV to mark which `status` properties reference the [binding data](#service-binding-schema):
     * The reference's `x-descriptors` with one-of:
       * ConfigMap:
-        * servicebinding:ConfigMap
+        * servicebinding:configMap
       * Secret:
-        * servicebinding:Secret
+        * servicebinding:secret
       * Individual binding items:
-        * servicebinding:Secret:host
-        * servicebinding:Secret:port
-        * servicebinding:Secret:uri
-        * servicebinding:Secret:`<binding_property>`  (where `<binding_property>` is any property from the binding schema)
+        * servicebinding:secret:host
+        * servicebinding:secret:port
+        * servicebinding:secret:uri
+        * servicebinding:secret:`<binding_property>`  (where `<binding_property>` is any property from the binding schema)
 
-2. Non-OLM Operator: - An annotation in the Operator's CRD to mark which `status` properties reference the binding data.  The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
+2. Non-OLM Operator: - An annotation in the Operator's CRD to mark which `status` properties reference the [binding data](#service-binding-schema).  The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
```

Comment:
Ideally, both `non-OLM Operator` and `OLM-enabled Operator` approaches should provide the same functionality. This would enable operator providers to be free to pick any which would make it extendible to non-OLD k8s.

And it looks like beside JSONPath/GO templates support, they already provide the same capability. Not sure how to add the templating to `x-descriptors` though

Reply:
I agree with providing the same functionality.  I was thinking that statusDescriptors (and I guess specDescriptors) could already describe any element, by using its `path` field - but I can't find documentation on whether that field supports JSON Path / Go template.  It seems the [OLM code](StatusDescriptor) is just handling it as a string?     

We could say in this spec that the `path` field supports those 2 templating styles, but I am wondering if that would break UI rendering in any way.    Not sure if @sbose78 came across this with SBO.

Reply:
Not entirely sure how/where the `path` field is used. This is tightly related to https://github.com/redhat-developer/service-binding-operator/issues/361